py3status/modules/imap.py

```diff
@@ -37,12 +36,23 @@ class Py3status:
     hide_if_zero = False
     imap_server = '<IMAP_SERVER>'
     mailbox = 'INBOX'
-    new_mail_color = ''
+    msg_unavailable = 'Mail: N/A'
```

Comment:
If it's ready, tag yellow `hold` for discussion regarding `msg_`, `format_`, `string_`.